spring-cloud-starter-stream-processor-image-recognition/src/test/resources/LICENSE

```diff
@@ -0,0 +1,203 @@
+Copyright 2015 The TensorFlow Authors.  All rights reserved.
```

Comment:
Why do we have this license file here in tests?
Maybe it should be presented in the project root at all?

Reply:
+1